Makefile

```
@@ -93,6 +93,37 @@ build:
+		for i in $$(seq 1 50); do \
+			pgrep -u $$(id -u) -x sdr-rs >/dev/null 2>&1 || break; \
+			sleep 0.2; \
+		done; \
+		if pgrep -u $$(id -u) -x sdr-rs >/dev/null 2>&1; then \
+			echo "error: sdr-rs did not exit within 10 s — close it and re-run make install"; \
+			rm -f $(RESTART_SENTINEL); \
+			exit 1; \
+		fi; \
+	fi
+
+install-bin: stop-running-app
```

checkmake (0.3.2)

[warning] Target "install-bin" defined multiple times (lines 93 and 125).

(uniquetargets)